README.md

```diff
@@ -1,3 +1,66 @@
-# slashml-python-client
+# SlashML Python client
+[SlashML](https://www.slashml.com/)
+# Introduction
+## Overview 
+This is a Python client (SDK) for SlashML. It allows the user to use the apps available and active in [SlashML-Dashboard](https://www.slashml.com/dashboard). 
+The apps can be together in the same code. For example, if the user wants to transcribe an audio file and get a summary, they can call speechtotext followed by summarization.
 
-to be updated to guide users
+State-of-the-art AI models from several service providers are available. At SlashML, we also do the benchmarking on these models for the user. This will give them an idea of the best service provider for their application. For the full list of the available models thru SlashML, click [here](##availlable-service-providers)
+
+
+
+to lets you run transcription jobs from your Python code or Jupyter notebook. The transcription can be done with three lines of code
+```
+import slashml
+
+speect_to_text = slashml.SpeechToText()
+transcribe_id= speect_to_text.transcribe(audio_url,service_provider="aws")
+status=speect_to_text.status(transcribe_id,service_provider=service_provider)
```

Comment:
This code will not work, you need to wait for the status to turn to `COMPLETED`.